examples/annotation_import/image.ipynb

```diff
@@ -20,12 +20,12 @@
       },
```

Comment:
<p>can we clear all the outputs for this? no need to show them what this looks like</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/523/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>